geopandas/plotting.py

```diff
@@ -260,6 +260,11 @@ def plot_series(s, cmap=None, color=None, ax=None, figsize=None, **style_kwds):
         fig, ax = plt.subplots(figsize=figsize)
         ax.set_aspect('equal')
 
+    if not s.shape[0]:
```

Comment:
While this is not too difficult to understand, I think `if s.empty:` would make the logic clearer, and I'm not aware of any advantage of checking this way (though perhaps I'm missing something?). Same for the dataframe check below.

Reply:
Yes, that could be a bit more readable.